pages.it/common/git-bisect.md

```diff
@@ -0,0 +1,20 @@
+# git bisect
+
+> Usa la ricerca binaria per trovare il commit che ha introdotto un bug.
+> Maggiori informazioni: <https://git-scm.com/docs/git-bisect>.
+
+- Avvia una sessione di bisect su un intervallo di commit limitato da un commit "cattivo", noto per contenere il bug, e un commit "buono", precedente al presentarsi del bug:
+
+`git bisect start {{commit_cattivo}} {{commit_buono}}`
+
+- Per ogni commit selezionato da `git bisect`, contrassegnalo con "bad" (cattivo) o "good" (buono) dopo averlo verificato:
+
+`git bisect {{good|bad}}`
+
+- Una volta che `git bisect` ha individuato il commit che ha introdotto il bug, termina la sessione di bisect e torna al ramo precedente:
+
+`git bisect reset`
+
+- Ignora un commit durante la sessione di bisect:
```

Comment:
here I skipped the translation because I thought it was a bit redundant. 
A question, then: must we translate 100% of the original version? How are we going to ensure the same consistency when changes are applied to the original version?

Reply:
@glenacota that's a very good question. Unfortunately, we have no good way of doing so, as of now :\. #2793 is probably our best option, but it's not going to be that simple.

Reply:
Also, there is no real need to be 100% accurate, but if we can while still producing an understandable and valid translation, that's better IMHO.

Reply:
as for your suggested addendum within parenthesis, I'm not entirely sure about it. My main concern is that it refers to "tests" that were never mentioned until then. If we want to leave an example of what could be a legit case for ignoring a bisect selection, shall we maybe rephrase it a description above like this?
`- Contrassegna ogni commit selezionato da `git bisect` come "bad" (cattivo) o "good" (buono) dopo averlo testato per verificare la presenza del bug:`
and
`- Ignora un commit durante la sessione di ricerca (ad esempio uno che fallisce i test per un motivo diverso dal bug ricercato):`

what do you think?

Reply:
- Contrassegna ogni commit selezionato da git bisect come "bad" (cattivo) o "good" (buono) dopo averlo testato per verificare la presenza del bug:

I like this, yes!

As for the second one, I would either say "durante la ricerca" or "durante la sessione di bisect". The first one is probably simpler.